scripts/shared/lib/deploy_operator

```diff
@@ -53,7 +55,20 @@ function subctl_install_subm() {
 }
 
 function install_subm_all_clusters() {
-    run_parallel "{1..3}" subctl_install_subm
+    if [[ $globalnet = "true" ]]; then
+        # When Globalnet is enabled, each cluster gets a non-overlapping globalCIDR from
+        # globalnet-cidr-range and this globalCIDR is stored in the clusterCRD for each
+        # cluster. When globalnet is deployed via subctl/Operator, during join operation,
+        # subctl first queries the existing clusters that are associated with the Broker
+        # and allocates a non-overlapping chunk of globalCIDR to the newly joining cluster.
+        # Due to this, we cannot install submariner globalnet (via subctl) in parallel and
+        # it has to be done sequentially.
+        run_parallel "1" subctl_install_subm
+        run_parallel "2" subctl_install_subm
+        run_parallel "3" subctl_install_subm
```

Comment:
You should use `with_context` command here in this case, also I'd stick it in a `for i in {1..3}`.
Alternatively you could add a `run_sequential` command for this in the utils if you wish.

Reply:
Okay, I'll add a new api - run_sequential in utils.
Also, with_context is used inside the run_parallel/sequential api.